src/ALE/MOM_remapping.F90

```diff
@@ -1889,8 +1889,8 @@ subroutine test_recon_consistency(test, scheme, n0, niter, h_neglect)
 
   enddo
 
-  write(label(1:8),'(i8)') niter
```

Comment:
Writing of quite large `niter` (>= 1e9) may produce out-of-memory access. `label`  should be enough long to keep -2**63. For example, `character(len=21) :: label`